packages/quantic/force-app/main/default/lwc/quanticTimeframeFacet/quanticTimeframeFacet.js

```diff
@@ -311,6 +312,10 @@ export default class QuanticTimeframeFacet extends LightningElement {
     this.initializeSearchStatusController(engine);
     this.initializeFacetController(engine);
     this.initializeDateFilterController(engine);
+    registerToStore(this.engineId, 'timeFrameFacets', {
```

Comment:
We should pass the formatting function as well to support custom timeframe labels.

Reply:
Since the `DateFilter` is registered using a different facet ID, we'd need to also register it in order to format the values properly. I guess the same thing goes for the custom filter of the Numeric facet.